lib/src/layer/polyline_layer/projected_polyline.dart

```diff
@@ -16,13 +16,16 @@ class _ProjectedPolyline<R extends Object> with HitDetectableElement<R> {
   _ProjectedPolyline._fromPolyline(Projection projection, Polyline<R> polyline)
       : this._(
           polyline: polyline,
-          points: List<DoublePoint>.generate(
-            polyline.points.length,
-            (j) {
-              final (x, y) = projection.projectXY(polyline.points[j]);
-              return DoublePoint(x, y);
-            },
-            growable: false,
-          ),
+          points: projection.projectList(polyline.points),
         );
+
+  /// Returns true if the points stretch on different versions of the world.
+  bool goesBeyondTheUniverse(double halfWorldWidth) {
```

Comment:
As much as I like this name, a declarative name would be more understandable in the future. 😄

Reply:
Agreed with this point, or at least a little more documentation (what is `halfWorldWidth`, why does it matter, where does it come from, etc.).

Reply:
agree